Intersect (Core)/Config/NpcOptions.cs

```diff
@@ -1,81 +1,83 @@
 ﻿namespace Intersect.Config
 {
-
     /// <summary>
     /// Contains configurable options pertaining to the way Npcs are handled by the engine.
     /// </summary>
     public class NpcOptions
     {
-
         /// <summary>
         /// Configures whether or not Npcs are allowed to reset after moving out of a specified radius when starting to fight another entity.
         /// </summary>
-        public bool AllowResetRadius = false;
+        public bool AllowResetRadius { get; set; } = false;
 
         /// <summary>
         /// Configures the radius in which an NPC is allowed to move after starting to fight another entity.
         /// </summary>
-        public int ResetRadius = 8;
+        public int ResetRadius { get; set; } = 8;
 
         /// <summary>
         /// Configures whether or not the NPC is allowed to gain a new reset center point while it is still busy moving to its original reset point.
         /// NOTE: Can be used to allow the NPCs to be dragged far far away, as it constantly resets the center of its radius!!!
         /// </summary>
-
-        public bool AllowNewResetLocationBeforeFinish = false;
+        public bool AllowNewResetLocationBeforeFinish { get; set; } = false;
 
         /// <summary>
         /// Configures whether or not the NPC should completely restore its vitals and statusses once it resets.
         /// </summary>
-        public bool ResetVitalsAndStatusses = false;
+        public bool ResetVitalsAndStatusses { get; set; } = false;
 
         /// <summary>
         /// Configures whether or not the level of an Npc is shown next to their name.
         /// </summary>
-        public bool ShowLevelByName = false;
+        public bool ShowLevelByName { get; set; } = false;
 
         /// <summary>
-        /// Configures whether or not the type of Npc is displayed adove to their name as a sprite tag.
-        /// [ Default is false ]
+        /// Configures whether or not to display Npc's Tags.
+        /// NOTE: Npc Tag Sprites are always loaded from the "tags" resource folder (rec size 32x20).
         /// </summary>
-        public bool ShowNpcTags = false;
-
-        /// <summary>
-        /// NpcTags above the Npc's Name Labels, only works if ShowNpcTags = true
-        /// [ Default is true ]
-        /// </summary> 
-        public bool NpcTagsOnTop = true;
+        public bool ShowTags { get; set; } = false;
 
         /// <summary>
-        /// NpcTags under the Npc's Name Labels, only works if ShowNpcTags = true
-        /// [ Default is false ]
+        /// Configures the position of the Npc Tags. Only works if ShowNpcTags = true.
+        /// 0: NpcTags above the Npc's Name Label.
+        /// 1: NpcTags under the Npc's Name Label.
+        /// 2: NpcTags as prefix (left) of the Npc's Name Label.
+        /// 3: NpcTags as suffix (right) of the Npc's Name Label.
         /// </summary>
-        public bool NpcTagsOnBottom = false;
+        public int TagsPosition { get; set; } = 0;
```

Comment:
Create an `enum` for this please.

```csharp
public enum TagPosition {
    None,
    Above,
    Below,
    Prefix,
    Suffix
}
```

Reply:
good idea! done 👍🏽

Reply:
Done 👍🏽 
altho, i didn't added a "None" enum, i don't think its required? if the dev doesn't wants to display tags at all, they can simply disable the feature instead. Thus, the default enum for now its "Adove".